### ISSUE TITLE
[DOCS] salt.modules.win_file

### ISSUE THREAD

**Issue by Selected6280** [saltstack/salt#63856]
**Description**
Please replace '\\' with '/' in the examples or use ''.

Example will fail:
![image](https://user-images.githubusercontent.com/124132746/224319824-5a8bc20a-3abb-4aa6-a361-bf306585e068.png)

Probably all examples on this site are affected

**Location or format of documentation**
https://docs.saltproject.io/en/latest/ref/modules/all/salt.modules.win_file.html#salt.modules.win_file.set_perms

**Comment by OrangeDog** [comment]
e.g. that example should be
```bash
salt '*' file.check_perms 'C:\Temp\' '{}' Administrators '{"Users": {"perms": "read_execute"}}'
```

**Comment by barbaricyawps** [comment]
@OrangeDog , does this mean we should probably close this issue? If so, let me know and I'll do that.

**Comment by OrangeDog** [comment]
No, it all needs fixing. Windows masters are still not supported as far as I know, and if the examples say they are for bash, they should be correct for bash.

**Comment by barbaricyawps** [comment]
Cool. I'll keep this open then.

**Comment by twangboy** [comment]
There is no "PowerShell" option for codeblocks... or at least there weren't at the time. So "Bash" is used.
